### PR TITLE
Bumped python-multipart to >=0.0.7

### PR DESCRIPTION
**Pull request by XChikuX**
## Types of Changes

- [x] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

*

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by botberry** [comment]
Thanks for adding the `RELEASE.md` file!



Here's a preview of the changelog:

---

Fix python-multipart version. There is a new 0.0.7 version to fix a Re-DoS bug



Here's the tweet text: 
```
🆕 Release (next) is out! Thanks to @Seetku for the PR 👏

Get it here 👉 https://beta.strawberry.rocks/release/(next)
```

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+Fix python-multipart version. There is a new 0.0.7 version to fix a Re-DoS bug
```
Comment (patrick91) [inline RELEASE.md:3]:
```suggestion
This releases updates the dependency of `python-multipart` to be at least `0.0.7` (which includes a security fix).

It also removes the upper bound for `python-multipart` so you can always install the latest version (if compatible) 😊
```

**Review by patrick91 — APPROVED** [review]
Perfect! Thanks!

**Comment by patrick91** [comment]
I've took the opportunity to update ruff and the config, since it was failing 😊

**Pull request merged**